input/data/config/default_config.json

```
@@ -1,6 +1,6 @@
 {
     "start_date": "2009:244",
-    "end_date": "2015:264",
+    "end_date": "2009:364",
```

```suggestion
    "end_date": "2015:264",
```
Is this change intentional? Or should we revert back to the previous default?

---

Good catch, thank you - just do this for testing to run shorter simulations.